### PR TITLE
Document_stopped_jobs and dependents for enqueue_many

### PR DESCRIPTION
**Pull request by eswolinsky3241**
Starting in version 1.15.0, enqueue_many supports dependent jobs using the depends_on argument. Additionally, jobs can now be enqueued with a new callback function, on_stopped, which executes when a currently executing job is stopped with the send_stop_job_command func.

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -249,6 +248,19 @@ def report_failure(job, connection, type, value, traceback):
 Failure callbacks are limited to 60 seconds of execution time.
 
 
+### Stopped Callbacks
+
+Stopped callbacks are functions that accept `job` and `connection` arguments.
+
+```python
+def report_stopped(job, connection):
```
Comment (selwin) [inline docs/docs/index.md:256]: Can you add an example of how to pass in `on_stopped` callback? Maybe put this on line 215.

Comment (eswolinsky3241) [reply docs/docs/index.md:257]: Added

File: docs/docs/index.md [hunk 2]
```diff
@@ -195,26 +194,27 @@ job_2 = queue.enqueue(say_hello, depends_on=dependency)
...
 To configure callback timeouts, use RQ's
 `Callback` object that accepts `func` and `timeout` arguments. For example:
 
 ```python
 from rq import Callback
 queue.enqueue(say_hello, 
               on_success=Callback(report_success),  # default callback timeout (60 seconds) 
-              on_failure=Callback(report_failure, timeout=10))  # 10 seconds timeout
+              on_failure=Callback(report_failure, timeout=10), # 10 seconds timeout
+              on_stopped=Callback(report_stopped, timeout=2m)) # 2 minute timeout
```
Comment (selwin) [inline docs/docs/index.md:217]: `2m` should be a string.

**Comment by selwin** [comment]
Thanks!

**Pull request merged**